src/main/resources/db/migration/V105__fix_hmp_wymott_offerings.sql

```diff
@@ -0,0 +1,2 @@
+update offering set withdrawn = true
+where organisation_id ='WMI'and course_id in (select c.course_id from course c where c.identifier ='NMS-AO');
```

Comment:
Can you make sure that the identifier is same in all environments, please

Just to be sure

Reply:
Yes, I checked this.